src/utils/AuthApi.ts

```diff
@@ -0,0 +1,12 @@
+import { useAxios, ApiHookResponse } from './UseAxios';
+import { HttpMethods } from './HttpMethods';
+
+const baseUrl = process.env.REACT_APP_API_URL;
+
+export const useAuthenticationApi = (
+  email: string,
+  password: string
+): ApiHookResponse =>
+  useAxios(HttpMethods.GET, `${baseUrl}/login`, {
```

Comment:
This looks great, what's your reasoning behind the HttpMethods enum? Seems a bit over engineered?

Reply:
I used `HttpMethods` to type the `httpMethod` parameter for `useAxios`. My goal was to prevent typos and to do less sanitization in `useAxios`. Would it be better to just accept a string and sanitize it in `useAxios`?